membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -230,6 +234,10 @@ object SupporterRatePlanToAttributesMapper {
       "2c92c0f955c3cf0f0155c5d9df433bf7",
       "2c92c0f955c3cf0f0155c5d9ddf13bc5",
       "2c92c0f955c3cf0f0155c5d9e2493c43",
+      // National Delivery
+      "8ad096ca8992481d018992a363bd17ad",
+      "8ad096ca8992481d018992a36256175e",
+      "8ad096ca8992481d018992a35f60171b",
```

Comment:
I know noone did that before, but in the test below each line has a comment saying which plan it is.  That might be handy here too!